### ISSUE TITLE
Training loss explodes

### ISSUE THREAD

**Issue by ruinianxu** [kuc2477/pytorch-deep-generative-replay#7]
I found that training loss for generator and critic will quickly explode and the accuracy can't achieve the one represented in the paper. Is it because unfit training hyperparameter? Btw, I modified the code to make it trainable with newer version pytorch.
